otter/tap/api.py

```diff
@@ -286,6 +289,21 @@ def on_client_ready(_):
     return s
 
 
+def setup_converger(parent, kz_client, dispatcher):
+    """Create a Converger service."""
```

Comment:
+ ", which has a Partitioner as a child service, so that if the Converger is stopped, the partitioner is also stopped"?